site/content/about/photos/2018-08-15.md

```diff
@@ -0,0 +1,7 @@
+---
+title: On the Frontlines of EdTech
+date: 2018-08-15
+link: https://photos.app.goo.gl/hQM9G7ssbjvvcBTZ8
+description: A night of networking and learning about what it means to be on the frontlines of EdTech
+image: https://lh3.googleusercontent.com/0ulxucEL1a5gG2fw5udYf_UlGhLbLZHhT3qhnUkhBHBhBFk3M9zLQDpBTL27Cl-GBCyuJukucwhJaCelKbrgWoIPD-sdt-gdDjGlaTplOQHrIZotHJY3HaB8sznlxVHmIBMWSr7Hncb-4QCqC9q3InNMi97Wj5TRrykJdVZfd1Ei9kqKCjjlnlrQg_ifRjKxksIJyaJhNvmDsOoQpAG9vDcH72Ricjv5S-NR8Zvy7mLW7F2FPSYfg8Bwhh7cpts1UanPNqBv1whjDcsnJ_hyS9nHb6k84ANuBJEEaO2sFFckpCF_Rpl8xOtwH8k6t6QIZVBV3Ke5v_n_599C_XGBQlo8GLX_AP18Tst9kVnBCMJVvPb_NAijslCF0Zc3tXAV-FNsT7kF9XERIISnL0KJexW1bBTlh8mVBO89j_krcoDqMkg_a6-ol2_YZOS7Mfymp2KbcCcYw9J934XcDmWbKE8KFSeGH5XUIVnhotQyfoAXY7ju-12HLFxYb-9mDotjjsiGUZfNSpl90nfpeMBeOO0yagYY92UI9Z8-Uhi0i6PjFICTmviayohuWC1eP_jzOsmRZ_7uBs97qSRCrsftPnUYdvxcOf2sdUPzHTVsww_BlHvy0Nv3zM2Bvji2fLCbaWIx8DGJ-9Cy9k7RA3Btq8o4M4J83td1pNzbTKLsWVg0eVRe4v0f258=w1840-h1228-no
```

Comment:
I actually did some research and turns out linking images from Google Photos like this doesn't last indefinitely so it would be better to use [Imgur](https://imgur.com) to host this cover image. We will need to change the other cover images as well but that can def be done in a future PR.

Reply:
Do you mean the album or the image? 

Google photos is a bit more profesh and the albums look nicer than imgur pics. 

For hosting photos we can also add them to this repo and link to them, maybe as attachments to an issue, or by adding them to the assets folder

Reply:
No I'm actually just talking about the banner image - we can't directly link images like that - the Google Photo direct image links expire after a certain time.

I like that all the images are hosted on Google Photos.

Reply:
This image just needs to be changed to an Imgur one and then I can merge! 🎉